test/validation/api/ipsec/ipsec.c

```diff
@@ -118,7 +146,7 @@ static void pktio_stop(odp_pktio_t pktio)
 		fprintf(stderr, "IPsec pktio stop failed.\n");
 
 	while (1) {
-		odp_event_t ev = recv_event(queue);
+		odp_event_t ev = recv_event(queue, EVENT_WAIT_TIME);
```

Comment:
I noticed that this now always takes a second, adding two seconds in the execution time of the inline in and out tests programs. recv_event(queue, 0) would not have changed the old behaviour. I wonder if the change is intentional? Maybe in some environments a non-zero wait time ensures that possible in-flight packets have time to end up in the queue, making non-zero wait time here reasonable?

Reply:
The non-zero wait time was not intentional here. Thanks for spotting this, will make it to zero wait time and update the PR.